connectors/sources/salesforce.py

```diff
@@ -29,7 +29,7 @@
 
 BASE_URL = "https://<domain>.my.salesforce.com"
 API_VERSION = "v58.0"
-TOKEN_ENDPOINT = "/services/oauth2/token"
+TOKEN_ENDPOINT = "/services/oauth2/token"  # noqa S105
```

Comment:
Wild that it thinks this is a password 🤷🏻

Reply:
Yeah I think the word "TOKEN" triggered it